### DISCUSSION TITLE
Fritzbox 7530 minidlna restarts router.

### DISCUSSION THREAD

**Discussion by ektorasdj** [Freetz-NG/freetz-ng#242] [Allgemein] [answered]
Hello i have freetz-ng latest trunk version and the only package i have installed is the minidlan 1.3.0. The last 3 trunk versions 
restart my fritzbox everytime i try to start the minidlna service and starts rebuilding the database. Same thing if i start to copy files through smb which i suspect the minidlna service is trying to discover the new files and then the router restarts again.  I havent change something ( i have 300 video files in the media server folder)  and i dont know if its the low ram- no swap on my 7530 (256mb ram) or the latest version is at fault.
Thanks for this tool and your support.

**Comment by fda77** [comment]
Can you attach a log/syslog of the crash?

**Reply by ektorasdj** [reply]
Yes I have attached minidlna log, and syslog.

[minidlna.log](https://github.com/Freetz-NG/freetz-ng/files/6232795/minidlna.log)

[logsys.txt](https://github.com/Freetz-NG/freetz-ng/files/6232796/logsys.txt)


in logsys at 2:22 the minidlna database starts rebuilding.
 I dont know if it can help you it doesnt seem to have something meaningful inside.
With stock fritzos it doesnt restart at all but i really want the minidlna.
Furthermore before the crash i have 55mb free ram and when the dlna rescan starts it jumps to 15mb free ram
*uploaded a new syslog maybe more helpful.

**Comment by fda77** [comment, marked as answer]
`[DECTDRV_ERR] If next time the error count exceeds the threshold value, a panic will be triggerd.`
Im not sure why this occurs, maybe your ram is low? Where do you save the cache of minidlna? In memory? And therefor it is rebuld every start? Im using onyl some mp3 and it still works for me

**Reply by ektorasdj** [reply]
I found the problem. Some video files where the culprit. I dont know why but those .mkv files made the router run out of memory when it was trying to read them. I deleted them and everything is back to normal.
The cache is in the external ssd and it doesnt get rebuild in every start.
Thank you and sorry for taking up your time while the problem came from my files.

**Comment by fda77** [comment]
minidlna uses many libs, maybe it helps to uses newer versions of them
https://github.com/Freetz-NG/freetz-ng/blob/master/make/minidlna/Config.in
